core/spec/integration/setup_spec.rb

```diff
@@ -30,6 +30,13 @@
       expect(tasks.users).to eql(users)
       expect(tasks.users.commands).to be_kind_of(ROM::CommandRegistry)
     end
+
+    it 'configures rom schema to store relations' do
+      users_schema = users_relation.schema
+      tasks_schema = tasks_relation.schema
+      expect(users_schema.relations[:users]).to eql(users_relation)
+      expect(tasks_schema.relations[:tasks]).to eql(tasks_relation)
+    end
```

Comment:
So, I understand that this makes sure we have finalized registry accessible via schemas, but there's also this change related to primary keys, and I'm not sure how that's related. Can we / should we have a separate spec example that checks primary key methods? ie canonical vs projected?